### PR TITLE
Snapshot description update.

### PR DESCRIPTION
**Pull request by twitchax**

### CODE REVIEW

**Comment by twitchax** [comment]
@yugangw-msft and @derekbekoe, if these are "Manage captured read-only disks", why are they not under `az disk snapshot`?

**Comment by yugangw-msft** [comment]
@twitchax, both are top level resources under "Microsoft.Compute" RP. Snapshot doesn't belong to disk, though there are relationships. Here are a few main properties:
1. You can create an empty snapshot, though it is not common.
2. Snapshot is mainly to support custom image, which is static.
3. You can create a snapshot by importing from another snapshot, a disk, or a blob uri
4. Once the snapshot is created, you can't change it any more
Maybe we can try new words like `Managed point-in-time copy of a managed disk, native blob, or another snapshot`?

**Comment by twitchax** [comment]
Sure, I like that suggestion best.

**Comment by derekbekoe** [comment]
@twitchax `another snapshots` -> `other snapshots` ?

File: src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py [hunk 1]
```diff
@@ -853,7 +853,7 @@
 
 helps['snapshot'] = """
     type: group
-    short-summary: Manage snapshots of Virtual Machines.
+    short-summary: Manage point-in-time copies of a managed disk, native blobs, or another snapshots.
```
Comment (tjprescott) [inline src/command_modules/azure-cli-vm/azure/cli/command_modules/vm/_help.py]: Recommended tweak: Manage point-in-time copies of managed disks, native blobs, or other snapshots.

**Comment by twitchax** [comment]
Fixed!

**Pull request merged**